requirements.txt

```diff
@@ -26,7 +26,7 @@ python-gnupg<0.4,>=0.3.7
 python-json-logger
 python-nmap<0.4,>=0.3.4
 pyyaml<4.0,>=3.11
-requests<3.0,>=2.7.0
+requests<3.0,>=2.7.0 [security]
```

Comment:
This is the right link:
https://pip.readthedocs.org/en/stable/reference/pip_install/#requirements-file-format

Should be `requests[security]<3.0,>=2.7.0` :)